src/pages/permissions/PermissionIdentityEditGroupBtn.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from "react";
+import { Button } from "@canonical/react-components";
+import { LxdIdentity } from "types/permissions";
+import usePanelParams from "util/usePanelParams";
+
+interface Props {
+  identities: LxdIdentity[];
+  className?: string;
+}
+
+const PermissionIdentityEditGroupsBtn: FC<Props> = ({
+  identities,
+  className,
+}) => {
+  const panelParams = usePanelParams();
+  const buttonText =
+    identities.length > 1
+      ? `Modify groups for ${identities.length} users`
```

Comment:
```suggestion
      ? `Modify groups for ${identities.length} identities`
```
Better stay consistent in naming things?